### PR TITLE
update contributing md file

### PR DESCRIPTION
**Pull request by LongfeiH**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] [local build the book](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [ ] [local build the slides](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [x] Unit test if appliable

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/ocademy-ai/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have applied tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: awesome/CONTRIBUTING.md [hunk 1]
```diff
@@ -70,6 +76,32 @@ Now, you are ready to submit a PR for your changes. Please make sure you have go
 
 TBD
 
+#### STEP 4. Resolve conflicts
+
+If there is no conflict in your pr, you can skip this step.
+Otherwise, you can resolve conflicts according to the following solution.
+
+1. go to github actions page, find your pr and copy the output of `Show database differences` and save it as a `.sql` file.
```
Comment (zhangqi444) [inline awesome/CONTRIBUTING.md:84]: PR and GitHub action must be capitalized.

File: awesome/CONTRIBUTING.md [hunk 2]
```diff
@@ -70,6 +76,32 @@ Now, you are ready to submit a PR for your changes. Please make sure you have go
...
+2. reset your code by `git reset --hard upstream/main`.
+3. execute the `.sql` file in SQLiteStudio.
+4. push your changes to the corresponding origin branch.
+
+**Note**: Do not pass the .sql file to pr together, you can delete it after you execute the file
```
Comment (zhangqi444) [inline awesome/CONTRIBUTING.md:89]: Missing period at the end

File: awesome/CONTRIBUTING.md [hunk 3]
```diff
@@ -70,6 +76,32 @@ Now, you are ready to submit a PR for your changes. Please make sure you have go
 
 TBD
 
+#### STEP 4. Resolve conflicts
+
+If there is no conflict in your pr, you can skip this step.
+Otherwise, you can resolve conflicts according to the following solution.
```
Comment (zhangqi444) [inline awesome/CONTRIBUTING.md:82]: Why two lines?

File: awesome/CONTRIBUTING.md [hunk 4]
```diff
@@ -70,6 +76,32 @@ Now, you are ready to submit a PR for your changes. Please make sure you have go
 
 TBD
 
+#### STEP 4. Resolve conflicts
```
Comment (zhangqi444) [inline awesome/CONTRIBUTING.md:79]: You can use GPT or Bard to help you refine the text.

File: awesome/CONTRIBUTING.md [hunk 5]
```diff
@@ -70,6 +76,32 @@ Now, you are ready to submit a PR for your changes. Please make sure you have go
...
+
+If there is no conflict in your pr, you can skip this step.
+Otherwise, you can resolve conflicts according to the following solution.
+
+1. go to github actions page, find your pr and copy the output of `Show database differences` and save it as a `.sql` file.
+2. reset your code by `git reset --hard upstream/main`.
```
Comment (zhangqi444) [inline awesome/CONTRIBUTING.md:85]: If each line ends with period, please capitalize the first letter.

**Review by zhangqi444 — APPROVED** [review]

**Review by zhangqi444 — APPROVED** [review]

File: awesome/CONTRIBUTING.md (hunk 1 above)
Comment (LongfeiH) [reply awesome/CONTRIBUTING.md]: done

File: awesome/CONTRIBUTING.md (hunk 3 above)
Comment (LongfeiH) [reply awesome/CONTRIBUTING.md]: done

File: awesome/CONTRIBUTING.md (hunk 4 above)
Comment (LongfeiH) [reply awesome/CONTRIBUTING.md:79]: done

File: awesome/CONTRIBUTING.md (hunk 2 above)
Comment (LongfeiH) [reply awesome/CONTRIBUTING.md]: done

**Pull request merged**